dotCMS/src/main/java/com/dotcms/publishing/manifest/ManifestBuilder.java

```diff
@@ -0,0 +1,15 @@
+package com.dotcms.publishing.manifest;
+
+import com.dotcms.publishing.manifest.ManifestItem.ManifestInfo;
+import com.dotmarketing.exception.DotRuntimeException;
+import java.io.Closeable;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public interface ManifestBuilder extends Closeable {
+     void create() throws IOException;
```

Comment:
I would avoid IOException in favor of RuntimeExceptions

Reply:
done https://github.com/dotCMS/core/pull/20632/commits/a69e6131a23032d5c1c7f424ecc96408152747d4